client/src/partials/templates/bhFindDocument.tmpl.html

```diff
@@ -142,15 +132,8 @@ <h4 class="media-heading">{{ doc.label }}</h4>
                 {{ doc.first + ' ' + doc.last }}
               </div>
               <div ng-show="$ctrl.session.showAction && $ctrl.selectedIndex === $index" class="document-actions">
-                <button
-                  title="{{ 'PATIENT_DOCUMENT.OVERVIEW' | translate }}"
-                  type="button"
-                  class="btn btn-sm btn-default">
-                  <i class="fa fa-eye"></i>
-                </button>
-
                 <a
-                  ng-href="/{{ doc.link }}" download="{{ doc.label }}"
+                  ng-href="/{{ doc.link }}" download="{{ doc.label + $ctrl.mimeIcon(doc.mimetype).ext }}"
```

Comment:
It's generally not a good idea to include functions in the view - they will be re-evaluated on every `$digest()` loop.

Could this be pre-processed in the controller?  Something like:
```js
// in the controller
documents.forEach(function (doc) {
  doc.downloadLink = doc.label + mimeIcon(doc.mimetype).ext;
});
```
.. in the view..
```html
<a ng-href="/{{doc.link}}" download="{{doc.downloadLink}}" ...></a>
```

That would be much more efficient!